### ISSUE TITLE
Recipe - Apply actionlint & shellcheck suggestions

### ISSUE THREAD

**Issue by aalmiray** [openrewrite/rewrite-github-actions#77]
[actionlint](https://github.com/rhysd/actionlint) is a static checker for GH Actions. Many of the problems identified by it could be solved by OpenRewrite rules. actionlint can also report problems identifiable by shellcheck.

Shellcheck rules could by provided independently from GH actions.

**Comment by sambsnyd** [comment]
Good idea. I don't know when we would prioritize creating something like this ourselves, but would be happy to accept it as a contribution.
